cloudify_agent/resources/script/linux.sh.template

```diff
@@ -73,8 +73,9 @@ package_url()
     if [[ ! -z "{{ conf.package_url }}" ]]; then
         echo "{{ conf.package_url }}"
     else
-        local distro="$(python -c 'import sys, platform; sys.stdout.write(platform.dist()[0].lower())')"
-        local distro_codename="$(python -c 'import sys, platform; sys.stdout.write(platform.dist()[2].lower())')"
+        local python_bin=$(command which python || command which python3 || echo "python")
```

Comment:
Why do you use command and which here?
Wouldn't it make more sense either to use 'command -v python' or just 'which python' (and the same for python3)?

Reply:
Also, defaulting back to 'python' after a lookup for the command 'python' has already fails seems like the wrong thing to do?

Reply:
Actually no reason for that we can use just `which python` and no need to fallback to python again

Reply:
@geokala I just noticed its used a lot in many places so if we go with `which` for example and `which` was not on the system would not be an issue ? which is the safer method ? I think `command -v ptyhon` ?